Source/Core/Core/AchievementManager.h

```diff
@@ -231,6 +232,8 @@ class AchievementManager
   std::chrono::steady_clock::time_point m_last_rp_time = std::chrono::steady_clock::now();
   std::chrono::steady_clock::time_point m_last_progress_message = std::chrono::steady_clock::now();
 
+  Config::ConfigChangedCallbackID m_hardcore_changed_callback_id;
```

Comment:
Since you never deregister the callback, you don't need this variable. It's fine to just call `Config::AddConfigChangedCallback` without storing the result anywhere.